server/utils/file/csvReader.js

```diff
@@ -60,14 +58,28 @@ const createReader = (filePath, onHeaders, onRow, onTotalChange) => {
       .on('error', reject)
   })
 
+  const calculateSize = () => new Promise((resolve, reject) => {
```

Comment:
do you really need this?

Reply:
yes, it's used in taxonomiesImportJob to better estimate the total items to be processed

Reply:
it is super fast when using the onTotalChange callback. I would suggest to remove it

Reply:
ok, I'll remove it